tests/core/pyspec/eth2spec/test/altair/epoch_processing/test_process_sync_committee_updates.py

```diff
@@ -40,7 +40,7 @@ def run_sync_committees_progress_test(spec, state):
     # Can compute the third committee having computed final balances in the last epoch
     # of this `EPOCHS_PER_SYNC_COMMITTEE_PERIOD`
     current_epoch = spec.get_current_epoch(state)
-    third_sync_committee = spec.get_sync_committee(state, current_epoch + 2 * spec.EPOCHS_PER_SYNC_COMMITTEE_PERIOD)
+    third_sync_committee = spec.get_sync_committee(state, current_epoch + 1)
```

Comment:
just noting: with the change to get the "next" sync committee in this PR, we only need to move to the next epoch (which happens to be in the next sync committee period) to get the third sync committee

Reply:
FYI we can add constant checks in spec builder `invariant_checks` once we decide the `ALTAIR_FORK_EPOCH`. (the stub value is `18446744073709551615` so it does not satisfy):
https://github.com/ethereum/eth2.0-specs/blob/39924c4f14d0702acc805f0d7799ee33dc81f364/setup.py#L409-L414